spec/models/claim/matching_attribute_finder_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+RSpec.describe Claim::MatchingAttributeFinder do
+  describe "#claim_ids_with_matching_attributes" do
```

Comment:
This is no longer the name of the method under test.

Reply:
I'm wondering if we could increase the confidence that these specs give us if we switched from using `include` to using `eq` instead and match the actual array of claims we expect. What do you think?